setup.py

```diff
@@ -650,6 +650,8 @@ def build_extensions(self):
         if get_platform() in ('win32', 'win-amd64'):
             # because Numpy 2.0 includes complex.h
             self._pyfftw_define_macros.append(("FFTW_NO_Complex", "1"))
+            # for import/export symbols correctly
```

Comment:
Is it really useful? Anyway, we cannot get version data on Windows?

Reply:
Apology my bad, this was something I thought it might be helpful on building wheel in Windows. It turns out this is only relevant when building the .dll. Reverted.